src/components/Progressbar/Progressbar.tsx

```diff
@@ -0,0 +1,75 @@
+import { Progress } from '@chakra-ui/react'
+import { Text } from '@chakra-ui/react'
+import style from './Progressbar.module.css'
+
+
+interface Props {
+    /**
+     * Цвет прогресс бара
+    */
+    colorScheme?: string,
+    /**
+     * Градиент прогресс бара
+    */
+    hasStripe?: boolean,
+    /**
+     * Анимированный прогресс бара
+    */
+    isAnimated?: boolean,
+    /**
+     * Максимальное значение прогресс бара
+    */
+    max?: number,
+    /**
+     * Минимальное значение прогресс бара
+    */
+    min?: number,
+    /**
+     * Размер прогресс бара
+    */
+    size?: 'sm' | 'md' | 'lg';
+    /**
+     * Значение прогресс бара
+     */
+    value?: number,
+    /**
+     * Выравнивание текста
+    */
+    alignText?: 'left' | 'right' | 'center';
+    /**
+     * Положение текста относительно прогресс бара
+    */
+    positionText?: 'top' | 'bottom' | 'hidden';
+    /**
+     * Подпись прогресс бара
+    */
+    text?: string;
+}
+
+/**
+ * Компонент параграфа
+ */
+export const Progressbar = (props: Props) => {
+    const { colorScheme = "blue", hasStripe=false, isAnimated=false, max=100, min=0, size='sm', value=50, alignText="center", text="text", positionText="hidden"} = props;
```

Comment:
Не обязательно задавать значения по-умолчанию. Компоненты Text и Progress вполне нормально смотрятся если их не передавать
```suggestion
    const { colorScheme, hasStripe, isAnimated, max, min, size, value, alignText, text, positionText } = props;
```